### ISSUE TITLE
Cannot match lowercase tags

### ISSUE THREAD

**Issue by lethefrost** [StarlaneStudios/vscode-comment-anchors#90]
https://github.com/ExodiusStudios/vscode-comment-anchors/issues/9#issue-363635240
I have seen this issue two years ago and it looks like you have added this feature? But it turns not to work for me... Even I manually added lowercase tags, the extension cannot parse and catch them. Thank you for any help, and this extension is really useful.

**Comment by eggachecat** [comment]
@lethefrost hi according [here](https://github.com/ExodiusStudios/vscode-comment-anchors/blob/f78b37dc41ce1df86f9782288c6fea2e19e89804/src/anchorEngine.ts#L328) it is seem to be intended

Maybe we can create a new parameter for this feature

**Comment by lethefrost** [comment]
> @lethefrost hi according [here](https://github.com/ExodiusStudios/vscode-comment-anchors/blob/f78b37dc41ce1df86f9782288c6fea2e19e89804/src/anchorEngine.ts#L328) it is seem to be intended
> 
> Maybe we can create a new parameter for this feature

Oh! I think you are right. There are a lot of `tag.toUpperCase()`. And I would be really appreciate if you add this feature!

**Comment by eggachecat** [comment]
@lethefrost I think you may just set  **Match case** to **false**, this may help a little (extension will catch and highlight it but still all-captial-case in the left view)

![image](https://user-images.githubusercontent.com/18111656/90388007-bfd1f100-e0b9-11ea-81c4-89a4be66adf9.png)

**Comment by lethefrost** [comment]
> @lethefrost Hi, my bad.
> 
> Is setting **Match case** to **false** can do the trick for you ?
> 
> ![image](https://user-images.githubusercontent.com/18111656/90388325-3373fe00-e0ba-11ea-9a35-da8f09982273.png)

Thank you! It works, now that the editor can catch the anchors regardless of cases. 
However what I want is a little bit more complicated, I hope it to match the case that exactly as I set in the settings.json, for example, if I have `todo`, `REVIEW`, `TODO` in the setting, then it will match `todo, TODO, REVIEW` but won't match `review` or `Review`. If I just simply set the `commentAnchors.tags.matchCase` to `false`, it will also get "review" as anchor which is undesirable because review is a real word and can occur in some nature language description, which is not intended to be an anchor.
